src/drivers/intel/ssp/mn.c

```diff
@@ -418,6 +418,29 @@ static inline int setup_initial_bclk_mn_source(uint32_t bclk, uint32_t *scr_div,
 	return 0;
 }
 
+/**
+ * \brief Reset M/N source clock for BCLK.
+ *	  If no port is using bclk, reset to use SSP_CLOCK_XTAL_OSCILLATOR
```

Comment:
how is this ensured?

Reply:
> how is this ensured?

The helper is_bclk_source_in_use() is used to ensure it.